### PR TITLE
The Very Big Supercomputer Commit

### PR DESCRIPTION
**Pull request by JuiceyBeans**
- Add recipes to supercompter
- Add maintenance hatch requirement
- Fix localisation of supercomputer + Naquadah Reactor I
- Replace Quartz Glass with Vibrant Quartz Glass in multiblock structure (as it is in Nomi)

Known issues:
- Maintenance hatch position is hardset to middle of top layer. If anyone knows how to let it be anywhere, that'd be great

### CODE REVIEW

**Review by LetterN — APPROVED** [review]

**Comment by JuiceyBeans** [comment]
Hang on don't merge yet, maintenance hatch is brokey

**Comment by JuiceyBeans** [comment]
Supercomputer is almost fully implemented now, just need to fix the NBT data for the data models in recipes

**Comment by JuiceyBeans** [comment]
In Nomi, pristine chance is 30%

File: kubejs/server_scripts/gregtech/supercomputer.js [hunk 1]
```diff
@@ -6,16 +6,35 @@ ServerEvents.recipes(event => {
...
-            // event.recipes.gtceu.simulation_supercomputer('kubejs:enderman')
-            //     .notConsumable(Item.of('hostilenetworks:data_model', '{data_model:{data:6,id:"hostilenetworks:enderman"}}'))
-            //     .itemInput('hostilenetworks:prediction_matrix')
-            //     .chancedOutput(Item.of('hostilenetworks:prediction', '{data_model:{id:"hostilenetworks:enderman"}}'), 5000, 500)
-            //     .Output(Item.of("hostilenetworks:prediction_enderman_prediction"))
-            //     .duration(512)
-            //     .EUt(10)
+    function supercomp(model, prediction, EUt) {
+        event.recipes.gtceu.simulation_supercomputer(`kubejs:${model}`)
+            .notConsumable(Item.of('hostilenetworks:data_model', '{data_model:{id:"hostilenetworks:' + model + '"}}')) // model nbts are still brokey
```
Comment (LetterN) [inline kubejs/server_scripts/gregtech/supercomputer.js]:
```suggestion
            .notConsumable(Item.of('hostilenetworks:data_model', '{data_model:{id:"hostilenetworks:' + model + '"}}').weakNBT()) // model nbts are still brokey
```

**Review by LetterN — CHANGES_REQUESTED** [review]

File: kubejs/server_scripts/gregtech/supercomputer.js (hunk 1 above)
Comment (JuiceyBeans) [reply kubejs/server_scripts/gregtech/supercomputer.js]: image
ITS PEAK, THANK YOU

**Pull request merged**